src/internal_events/common.rs

```diff
@@ -150,14 +150,13 @@ impl<'a, const INTENTIONAL: bool> InternalEvent for ComponentEventsDropped<'a, I
 
 #[derive(Debug)]
 pub struct SinkRequestBuildError<E> {
-    pub name: &'static str,
     pub error: E,
 }
 
 impl<E: std::fmt::Display> InternalEvent for SinkRequestBuildError<E> {
     fn emit(self) {
         error!(
-            message = format!("Failed to build request for {}", self.name),
+            message = format!("Failed to build request"),
```

Comment:
Should it end in a period ? It seems like most do.

Reply:
It _didn't_ before which I thought was somewhat odd? It passes the `check-events`, but can add it in regardless.

Reply:
Resolved in https://github.com/vectordotdev/vector/pull/14510/commits/a4600105f22e9b7b13ff262740a5067ae490cf14